Chat_Log/Settings.cs

```diff
@@ -0,0 +1,7 @@
+namespace Chat_Log
+{
+    public class Settings
+    {
+        public int MaxLogMessages { get; set; } = 300;
+    }
+}
```

Comment:
This is the settings file, when creating settings the first time it will have 300 by default but after that it will use the value the server owner choose